### ISSUE TITLE
Asus u4100e i7-1165G7 can't found hdd.

### ISSUE THREAD

**Issue by git2wing** [deniro98/hackintosh-asus-zenbook-duo-ux482ea#7]
I use an asus u4100e with i7-1165G7 16G 512G MX450, which should be an upgrade to the ux482 .
and start bigsur installation recovery mode successfully. However, the primary hard drive could not be found. 
My computer uses Intel VMD drive (Intel Volume Management Device technology) under windows.
Does your ux482 also have an intel vmd controller?  Can you give me some advice or guidance?

**Comment by djdeniro** [comment]
Hi, you need to replace you NVME.

**Comment by chundk** [comment]
> Hi, you need to replace your NVME.

no, I had the same issue, The reason why no hard driver can' be found was the Intel VMD feature is enabled.

Just disable the VMD feature via BIOS. HDD should be found1

**Comment by chundk** [comment]
> I use an asus u4100e with i7-1165G7 16G 512G MX450, which should be an upgrade to the ux482 . and start bigsur installation recovery mode successfully. However, the primary hard drive could not be found. My computer uses Intel VMD drive (Intel Volume Management Device technology) under windows. Does your ux482 also have an intel vmd controller? Can you give me some advice or guidance?

Mac OS doesn't support such features, raid vmd...
